src/ooxml/java/org/apache/poi/openxml4j/opc/ZipPackage.java

```diff
@@ -239,8 +239,8 @@ private static ZipEntrySource openZipEntrySourceStream(ThresholdInputStream zis)
         }
 
         if (this.zipArchive == null) {
-            return this.partList.values().toArray(
```

Comment:
Whoops! Yeah that's actually how it's done further down the method. I'll make the change and ammend the pull request